dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -0,0 +1,291 @@
+#!/bin/bash
+
+set -e
+
+init() {
+#OS specific defaults
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    DEFAULT_OC_PUBLIC_HOSTNAME="192.168.65.2"
+    DEFAULT_OC_PUBLIC_IP="192.168.65.2"
+    [ -z "$CHE_OAUTH_GITHUB_CLIENTID" ] && export CHE_OAUTH_GITHUB_CLIENTID=d210c15f1a0c026a808c;
+    [ -z "$CHE_OAUTH_GITHUB_CLIENTSECRET" ] && export CHE_OAUTH_GITHUB_CLIENTSECRET=cc2589e6eed89ca460c8dbd13b9342cfddbe8098;
+else
+    DEFAULT_OC_PUBLIC_HOSTNAME="127.0.0.1"
+    DEFAULT_OC_PUBLIC_IP="127.0.0.1"
+    [ -z "$CHE_OAUTH_GITHUB_CLIENTID" ] && export CHE_OAUTH_GITHUB_CLIENTID=23638e66bf0d53bdc777;
+    [ -z "$CHE_OAUTH_GITHUB_CLIENTSECRET" ] && export CHE_OAUTH_GITHUB_CLIENTSECRET=e20edd408a4becda5f6ccb83abb4b1e7a432fe46;
+fi
+export OC_PUBLIC_HOSTNAME=${OC_PUBLIC_HOSTNAME:-${DEFAULT_OC_PUBLIC_HOSTNAME}}
+export OC_PUBLIC_IP=${OC_PUBLIC_IP:-${DEFAULT_OC_PUBLIC_IP}}
+
+DEFAULT_CHE_MULTIUSER="false"
+export CHE_MULTIUSER=${CHE_MULTIUSER:-${DEFAULT_CHE_MULTIUSER}}
+
+DEFAULT_OPENSHIFT_USERNAME="developer"
+export OPENSHIFT_USERNAME=${OPENSHIFT_USERNAME:-${DEFAULT_OPENSHIFT_USERNAME}}
+
+DEFAULT_OPENSHIFT_PASSWORD="developer"
+export OPENSHIFT_PASSWORD=${OPENSHIFT_PASSWORD:-${DEFAULT_OPENSHIFT_PASSWORD}}
+
+DEFAULT_OPENSHIFT_NAMESPACE_URL="eclipse-che.${OC_PUBLIC_IP}.nip.io"
+export OPENSHIFT_NAMESPACE_URL=${OPENSHIFT_NAMESPACE_URL:-${DEFAULT_OPENSHIFT_NAMESPACE_URL}}
+
+DEFAULT_OPENSHIFT_FLAVOR="ocp"
```

Comment:
I think this script should support only deploying with `OCP` flavor, am I wrong?

Reply:
you are right Sergio

Reply:
done